draft-add-requirements.md

```diff
@@ -142,18 +271,99 @@ Narrative required.
 
 ### Private enterprise names
 
-Narrative required.
+As stated above, an enterprise-managed Wi-Fi network might be able to access both the Internet and a private
+intranet. The private domains managed by the enterprise might only be resolvable using a specific DNS resolver,
+hence use of that resolver is essential for such domains. However it does not necessarily mean that all queries
+for all domains have to flow through that resolver.
+
+Only sending the necessary queries through the enterprise resolver, and not generic Internet queries, has the privacy
+benefit of only exposing traffic to the enterprise that fall within a limited set of domains.
+
+Using encrypted DNS for private names also opens up the possibility of doing private name resolution outside
+of the content of a VPN or managed network. If the DNS resolver authenticates clients, it can offer its resolver
+for private names on a publicly accessible server, while still limiting the visibility of the DNS traffic.
 
 | Requirement | Description |
 | R11.1 | Example requirement |
 
-## Discover a mapping between a public domain and a resolver {#last-usecase}
+## Encrypted resolvers for content providers {#cdn-content}
 
-Narrative required.
+Content Delivery Networks (CDNs), and content-providers more broadly, can also provide encrypted DNS resolvers that can
+be used by clients over the public Internet. These resolvers can either allow resolution of all public names (like
+normal recursive resolvers), or be designed to serve a subset of names managed by the content provider (like an
+authoritative resolver). Using these resolvers can allow the content provider to directly control how DNS answers are
+used for load balancing and address selection, which could improve performance of connections to the content provider.
+
+Using a content-provider's encrypted resolver can also provide several privacy and security benefits: 
+
+- Prevent devices along the network path to the content-provider's resolver from observing client DNS messages
+- Verify that answers come from the entity that manages the domains being resolved
+- Reduce the number of entities able to monitor the specific names accessed by a client to only the client and the content provider, assuming that the content provider would already see the names upon a secure connection later being made based on the DNS answers (e.g., in the TLS SNI extension)
+
+However doing so can create other issues. (To be described)
```

Comment:
```suggestion
```

Reply:
I don't support putting in an unexplained and unjustified sentence here to undercut this use case. These explanations need to be objective.

Reply:
To explain, this was intended to go into (completely objectively) the consequences implied by use of #cdn-content, e.g. bypass of intended monitoring or filtering. Such actions can be a benefit or a drawback, potentially both at the same time. I agree the sentence can't remain without justification, so I'll delete for -00.